db/data_migration/20170908110927_republish_consultations_with_topical_events.rb

```diff
@@ -0,0 +1,6 @@
+document_ids = Consultation.find_each.map { |c| c.document_id if c.topical_events.present? }.compact
```

Comment:
You might want to `uniq` these too... Probably not a big issue as there aren't that many but you'll get duplicate document ids in here I think.